kroxylicious-operator/DEV_GUIDE.md

```diff
@@ -0,0 +1,82 @@
+This is the Kroxylicious operator for Kubernetes.
+
+# Hacking and Debugging
+
+If you want to iterate quickly on the operator the simplest way is to run it as a process on your host (i.e. *not* running it within a Kubernetes cluster).
+
+Note: The Integration Tests will only run if your kubectl context is pointing at a cluster. For development, we recommend using  `minikube`, for example:
+
+```bash
+minikube start --kubernetes-version=latest --driver=podman 
+````
+
+You should now be able to run the tests using `mvn`.
+
+If you want to run the `OperatorMain` (e.g. from your IDE, maybe for dubugging) then you'll need to install the CRD:
+
+```bash
+kubectl install -f src/main/resources/META-INF/fabric8
```

Comment:
fails for me with:

```
kubectl install -f src/main/resources/META-INF/fabric8
error: unknown command "install" for "kubectl"
```

should this be `kubectl create`?

Reply:
`apply`

Reply:
`create` will also work, but `kubectl` will get shirty with you if you then try to use `apply` later